samples/host_verify/README.md

```diff
@@ -0,0 +1,68 @@
+
+# The host verification sample
+
+- Demonstrates the attestation of a remote host
+- Requires only the `openenclave/host_verify.h` header, as well as some C standard header files
+- Also runs in simulation mode and when both BUILD_ENCLAVES and HAS_QUOTE_PROVIDER flags are turned off
+
+Prerequisites: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+
+## About the host verification sample
+
+When a host is doing host verification, or remote host attestation to be exact, it means that it is trying to authenticate its hardware and software settings so that it can determine whether or not to trust the remote host.
+This sample does remote host attestation by taking either of the two arguments:
+
+- An SGX report
+- An SGX certificate
+
+A user can generate a report or certificate with `oecert`. See [here](https://github.com/openenclave/openenclave/blob/master/tests/tools/oecert/README.md) for more details.
+
+In the main function, if it sees the attribute `-r`, then it will continue to read the next argument to get the report and call `verify_report()` to verify the report.
+Likewise, if it sees the attribute `-c`, then it will continue to read the next argument to get the certificate and call `verify_cert()` to verify the certificate.
+There is also an `enclave_identity_verifier()` function, which is called by `verify_cert()` and shows the information of a certificate if you feed it to the program.
+
+## Build and run
+
+Open Enclave SDK supports building the sample on both Linux and Windows.
+Linux supports two types of build systems, GNU Make with `pkg-config` and CMake,
+while Windows supports only CMake.
+
+### Linux
+
+#### CMake
+
+This uses the CMake package provided by the Open Enclave SDK.
+
+```bash
+cd host_verify
+mkdir build && cd build
+cmake ..
+make run
+```
+
+#### GNU Make
+
+```bash
+cd host_verify
+make build
+make run
+```
+
+### Windows
+
+#### CMake
+
+```bash
+mkdir build && cd build
+cmake .. -G Ninja -DNUGET_PACKAGE_PATH=C:\oe_prereqs
+ninja
+ninja run
+```
+
+When you see the following message displayed on the screen, it means you have successfully run the sample.
+
+```bash
+Usage:
+  ./host_verify -r <report_file>
+  ./host_verify -c <certificate_file>
```

Comment:
Do we have tools/other samples that can generate a report and certificate? If so we should add to the document how people can generate these files and run the sample.

Reply:
I only know about the `oecert` tool so I include the link in the middle of the README file.